.github/workflows/test_and_make_release.yml

```diff
@@ -12,24 +12,131 @@ on:
       - master
 
 jobs:
-  test:
+  template_test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.8, 3.7, 3.6]
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install tools (Poetry)
+        uses: Gr1N/setup-poetry@v2
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install tox
+          pip install cookiecutter
```

Comment:
Why using 2 different syntaxes (`python -m pip` and `pip`) ?

Reply:
Good point ! I did a straight copy paste from https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template, which does exactly that... Any clue why they wrote it this way?

Reply:
anyway, remove `python -m `.

Reply:
No clue